IWXXM/WAFSSigWxFC.xsd

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<schema elementFormDefault="qualified" targetNamespace="http://icao.int/iwxxm/2021-2" version="1.0.0RC1" xmlns:iwxxm="http://icao.int/iwxxm/2021-2" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:aixm="http://www.aixm.aero/schema/5.1.1" xmlns="http://www.w3.org/2001/XMLSchema">
+<schema elementFormDefault="qualified" targetNamespace="http://icao.int/iwxxm/2021-2" version="1.0.0RC2" xmlns:iwxxm="http://icao.int/iwxxm/2021-2" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:aixm="http://www.aixm.aero/schema/5.1.1" xmlns="http://www.w3.org/2001/XMLSchema">
 	<include schemaLocation="metFeature.xsd"></include>
 	<import namespace="http://www.aixm.aero/schema/5.1.1" schemaLocation="http://www.aixm.aero/schema/5.1.1/AIXM_Features.xsd"></import>
 	<import namespace="http://www.opengis.net/gml/3.2" schemaLocation="http://schemas.opengis.net/gml/3.2.1/gml.xsd"></import>
```

Comment:
I apologize @blchoy for not catching these typos in RC1.
Line 9: weakly-typed, not week-type
Line 180: replace "Oceanic and Atmospheric Administration (NOAA)" with "Weather Service (NWS)"

Reply:
Line 9: Fixed.
Line 180: I actually get this from WMO Common Code Table C-1 (http://codes.wmo.int/common/centre/177) which is being used in existing BUFR SIGWX files.  As we are not making direct reference to this table I see no reason why we cannot change.  Updated in commit https://github.com/wmo-im/iwxxm/commit/14b70a04a0e06106377cd927d6dfecf9ca62f08f

Reply:
I stumbled into NOAA vs NWS politics. 😁